README.md

```diff
@@ -104,7 +81,6 @@ Click the button below to start deployment.
 
 1. To start a deployment, provide the following required template parameters, and then click the `Purchase` button:
    - **Application Name** - Type the name of the log source to appear in the Alert Logic console.
-   - **Storage Name** - Type any storage account name (that does not currently exist).
```

Comment:
Some of the other parameters are no longer needed either, no? Service Principal ID, App Client ID, App Client Secret.